test/pkg.jl

```diff
@@ -533,4 +533,19 @@ temp_pkg_dir() do
         @test ret === nothing && out == ""
         @test contains(err, nothingtodomsg)
     end
+
```

Comment:
On the other hand hijacking that test does not seem like a good idea after all. But since there is precedent for modifying metadata in the test suite should I leave the test in?